serverlessworkflow/modules/ROOT/pages/getting-started/create-your-first-workflow-service.adoc

```diff
@@ -1 +1,274 @@
-= Create your first workflow service
\ No newline at end of file
+= Create your first workflow service
+
+Learn how to create a Hello World Serverless Workflow application. This guide covers:
+
+* Bootstrapping an application
+* Creating a workflow
+* Running the application
+* Testing the application
+
+== Prerequisites
+
+To complete this guide, you need:
+
+* https://adoptium.net/[JDK 11+] installed with `JAVA_HOME` configured appropriately
+* https://maven.apache.org/install.html[Apache Maven]
+* https://quarkus.io/guides/cli-tooling[Quarkus CLI]
+
+To edit your workflows:
+
+* Visual Studio Code with https://marketplace.visualstudio.com/items?itemName=redhat.java[Red Hat Java Extension]
+and https://marketplace.visualstudio.com/items?itemName=redhat.vscode-extension-serverless-workflow-editor[Red Hat Serverless Workflow Editor]
+installed
+
+== Architecture
+
+In this guide, we'll create a Serverless Workflow application that serves a `hello_world` endpoint.
+The workflow comprises two states:
+
+. `Inject Hello World` - Injects a `Hello World` message into the response
+. `Inject Mantra` - Injects a `Mantra` message into the response
+
+image::getting-started/hello-world-workflow.png[]
+
+== Solution
+We recommend that you follow the instructions from <<boostrapping-the-project,Bootstrapping the Project>>
+and onwards to create the application step by step.
+
+However, you can go right to the completed example by cloning the git repository:
+
+The solution is located in the `kogito-quarkus-examples/serverless-workflow-hello-world`
+https://github.com/hbelmiro/kogito-examples/tree/main/kogito-quarkus-examples/serverless-workflow-hello-world[directory].
+
+[[boostrapping-the-project]]
+== Bootstrapping the Project
+
+Run the following command in a terminal window to create the project skeleton:
+
+[source,shell]
+----
+quarkus create app -x=kogito-quarkus-serverless-workflow,quarkus-container-image-jib,quarkus-resteasy-jackson,quarkus-smallrye-openapi org.kie.kogito.examples:serverless-workflow-hello-world:1.0
```

Comment:
What about providing also the quarkus-maven-plugin command? Then we have to care about one prerequisite less.

Reply:
Please also separate this command in separated lines with `\`:

```suggestion
quarkus create app \
    -x=kogito-quarkus-serverless-workflow \
    -x=quarkus-container-image-jib \
    -x=quarkus-resteasy-jackson \
    -x=quarkus-smallrye-openapi \
    org.kie.kogito.examples:serverless-workflow-hello-world:1.0
```

Also, please explain what we are doing here. This way, newcomers will understand what they are doing rather than just copying and pasting.